sheetScript.gs

```diff
@@ -583,24 +583,21 @@ function removeShortsFilter(videoId) {
     id: videoId,
   });
   if (response.items && response.items.length && response.items[0].contentDetails.duration) {
-    return !isLessThanAMinute(response.items[0].contentDetails.duration)
+    return !isLessThanThreeMinutes(response.items[0].contentDetails.duration)
   }
   return false;
 }
 
 // Checks if an ISO 8601 duration is less or equal than a minute.
```

Comment:
Can you update this comment too?

Reply:
Is GitHub not showing the right thing? Line 586 is definitely edited and there is no comment there.

Reply:
Sorry, I meant line 591, the last line of the snippet. GitHub UI could've made it more clear

Reply:
I could have sworn that I already did that. I did a Ctrl+F for "minute" and updated it all. Anyway, now it is done.

Reply:
Thanks!